### PR TITLE
Reconcile PR #440 with new fhicl file format, include CRT in stage 0

### PR DESCRIPTION
**Pull request by SFBayLaser**
This PR is an attempt to reconcile PR #440 which is Andrea Scarpelli's updates to the optical timing but which was based on an older version of icaruscode, hence incompatible with the updated fhicl files for production running. Also, we are working on developing a PMT/CRT filter and this PR also includes updates to include running the current CRTPMT ana module. 
I recommend this is vetted by PMT experts first... I'm still not reproducing the results shown by the CRT folks with the PMT/CRT time differences...

### CODE REVIEW

**Comment by SFBayLaser** [comment]
trigger build

File: fcl/reco/Stage1/Run2/stage1_run2_icarus.fcl [hunk 1]
```diff
@@ -5,7 +5,7 @@ process_name: stage1
 physics.reco: [ flashfilter,
                 @sequence::icarus_filter_cluster3D,
                 @sequence::icarus_pandora_Gauss,
-                @sequence::icarus_reco_fm,
+                #@sequence::icarus_reco_fm,
```
Comment (PetrilloAtWork) [inline fcl/reco/Stage1/Run2/stage1_run2_icarus.fcl:8]: Why is flash matching being removed from Stage1?

File: icaruscode/IcarusObj/PMTWaveformTimeCorrection.h [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
  * @brief  Holds the event-by-event waveform timing adjustment.
  * @author Andrea Scarpelli (ascarpell@bnl.gov)
  * @date   March 23 2021
  * @see    icaruscode/Timing/DataProducts/PMTWaveformTimeCorrection.cxx
  */
 
-#ifndef ICARUSCODE_TIMING_DATAPRODUCTS_PMTWAVEFORMTIMECORRECTION_H
-#define ICARUSCODE_TIMING_DATAPRODUCTS_PMTWAVEFORMTIMECORRECTION_H
+#ifndef ICARUSCODE_ICARUSOBJ_PMTWAVEFORMTIMECORRECTION_H
+#define ICARUSCODE_ICARUSOBJ_PMTWAVEFORMTIMECORRECTION_H
```
Comment (PetrilloAtWork) [inline icaruscode/IcarusObj/PMTWaveformTimeCorrection.h:10]: Nice, but you missed the comment on `#endif` at the end of the file.

**Review by PetrilloAtWork — COMMENTED** [review]
I am reviewing only the last commit from @SFBayLaser.
I can't really tell about the "changes" to CRT, and GitHub is a hell of a tool for this type of reviews.
The only question is about the removal of the flash matching from Stage1, which I am not sure what is driven by. I bet it's my lack of understanding, but please just confirm so.

File: fcl/reco/Stage1/Run2/stage1_run2_icarus.fcl (hunk 1 above)
Comment (SFBayLaser) [reply fcl/reco/Stage1/Run2/stage1_run2_icarus.fcl:8]: Fallout... that sequence is running the "simple flash match" which got broken when some updates were made in sbncode level stuff but the corresponding updates at the icaruscode level were not included. I'll try turning that back on...

Comment (SFBayLaser) [reply fcl/reco/Stage1/Run2/stage1_run2_icarus.fcl:8]: I think still broken... I _**think**_ this is tied to PR #462...

**Comment by SFBayLaser** [comment]
I updated the base to the current develop branch, consistent with v09_60_01... but still having some trouble reproducing CRT/PMT matching results. Need a CRT expert to try running to see what is missing...

**Comment by rennney** [comment]
trigger build

**Comment by SFBayLaser** [comment]
trigger build

**Comment by mmrosenberg** [comment]
trigger build

**Pull request merged**